snippets/csharp/Messages.cs

```diff
@@ -0,0 +1,50 @@
+using Breez.Sdk;
+
+public class MessagesSnippets
+{
+    public void SignMessage(BlockingBreezServices sdk)
+    {
+        // ANCHOR: sign-message
+        var message = "<message to sign>";
+        try
+        {
+            var signMessageResponse = sdk.SignMessage(new SignMessageRequest(message));
```

Comment:
Having a separate variable for request object may make things more clear.

Note: If approved, this change should be applied for all the bindings.

Reply:
Fixed [fd6678f](https://github.com/breez/breez-sdk-docs/pull/185/commits/fd6678f887ee34c7401dee84db39d55c994d2faa)